topics/matlab/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: matrix-laboratory
+created_by: Cleve Moler
+display_name: MATLAB
+logo: matlab.png
+released: 1984
+short_description: MATLAB is a programming platform for numerical computation and visualization.
+topic: matlab
+url: http://mathworks.com/products/matlab
+wikipedia_url: https://en.wikipedia.org/wiki/MATLAB
+---
+MATLAB is a high-performance language developed by MathWorks for technical computing, visualization, and programming. It is written in C, C++, Java and available for the various operating system such as Windows, macOS, and Linux.Generally used by engineers and scientists in industry and academics for data analysis, signal processing, optimization and many other types of scientific computations.
```

Comment:
- `for the various operating system such as Windows, macOS, and Linux` change to just `for Windows, macOS, and Linux`
- `Linux.Generally` can you stick a space after `.`.

Thanks!